doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,51 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Log Management Service (LMS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Log Management Service (LMS).
+
++-------------+-----------------------------------------------------------------------+---------------------------------------+
+| Reference   | Requirement                                                           | Notes                                 |
++=============+=======================================================================+=======================================+
+| pas.lms.001 | LMS must support log management from multiple, distributed sources    |                                       |
++-------------+-----------------------------------------------------------------------+---------------------------------------+
+| pas.lms.002 | LMS must manage log rotation at configurable time periods             |                                       |
++-------------+-----------------------------------------------------------------------+---------------------------------------+
+| pas.lms.003 | LMS must manage log rotation at configurable log file status (%full)  |                                       |
++-------------+-----------------------------------------------------------------------+---------------------------------------+
+| pas.lms.004 | LMS must manage archival and retention of logs for configurable       |                                       |
+|             | time periods by different log types                                   |                                       |
++-------------+-----------------------------------------------------------------------+---------------------------------------+
+| pas.lms.005 | LMS should ensure log file integrity                                  |                                       |
```

Comment:
pas.lms.005; LMS shoudl ensure log file integrity: what does "integrity" mean in this context?